images/dnsmasq/Makefile

```diff
@@ -13,19 +13,19 @@
 # limitations under the License.
 
 VERSION ?= $(shell git describe --tags --always --dirty)
-REGISTRY ?= gcr.io/google_containers
+REGISTRY ?= gcr.io/google-containers
```

Comment:
yeah, they're basically equivalent as far as docker is concerned, but various utilities prefer the real project name (`google-containers`) instead of the docker hack (`google_containers`), so I've been slowly updating references as I come across them.

one of these days I might just do a large fixit.